lib/src/modules/apply.cc

```diff
@@ -206,17 +336,20 @@ exit exit_code
     {
         module_name = "apply";
         actions.push_back(apply_ACTION);
+        actions.push_back(prep_ACTION);
+        PCPClient::Schema apply_input_schema { apply_ACTION, lth_jc::JsonContainer { apply_ACTION_INPUT_SCHEMA } };
+        input_validator_.registerSchema(apply_input_schema);
 
-        PCPClient::Schema input_schema { apply_ACTION, lth_jc::JsonContainer { apply_ACTION_INPUT_SCHEMA } };
-        input_validator_.registerSchema(input_schema);
+        PCPClient::Schema prep_input_schema { prep_ACTION };
```

Comment:
Is this how `PCPClient::Schema` is intended to work? This is the first built-in cpp module with multiple module actions?

Reply:
This all looks correct to me